README.md

```diff
@@ -1,2 +1,7 @@
-# ruby-homework
-A repository I will use to solve my Ruby homework
+# ruby-homework-modified
+A repository I will use to solve my Ruby homework from ruby course [https://github.com/monorkin/learn.rb](URL)
+
+##My favorite programming languages
+*C#*, **python**, *java*
+
+[westi](https://wallpapercave.com/wp/LtQcp1z.jpg.png)
```

Comment:
Image preview doesn't work because link is pointing to 404 page.

Reply:
Forgot to mention, add `!` in front for the preview to work.
I haven't specified this in the assignment and you've seen how the review process works so I'll give you approval regardless.